mock/server/authentication.go

```diff
@@ -73,3 +73,17 @@ func getAuthenticateFromToken(c *gin.Context) {
 
 	c.JSON(http.StatusOK, body)
 }
+
+// validateToken returns mock response for a http GET.
+//
+// Don't pass "Token" in header to receive an error message. Pass "0" in "Token" header to receive invalid token resp.
+func validateToken(c *gin.Context) {
+	err := "error"
+
+	token := c.Request.Header.Get("Token")
```

Comment:
should this mock be checking the "Authorization" header rather than "Token"? 

https://github.com/go-vela/server/pull/790/files#diff-150d061ec4206a8396c1e267a3493c43049fb93cca6b50cb99cf4fa8c1adc2e7R93